nvme-print.c

```diff
@@ -1614,6 +1614,103 @@ void nvme_show_endurance_group_event_agg_log(
 	}
 }
 
+void json_lba_status_log(struct nvme_lba_status_log_page *lba_status)
+{
+	struct json_object *root;
+        struct json_object *desc;
+        struct json_object *element;
+	struct json_array *desc_list;
+        struct json_array *elements_list;
+	__u32 num_lba_desc, num_elements;
+
+	root = json_create_object();
+	json_object_add_value_uint(root, "lslplen",
+		le32_to_cpu(lba_status->lslplen));
+        num_elements = le32_to_cpu(lba_status->nlslne);
+	json_object_add_value_uint(root, "nlslne",
+		num_elements);
+	json_object_add_value_uint(root, "estulb",
+		le32_to_cpu(lba_status->estulb));
+	json_object_add_value_uint(root, "lsgc",
+		le16_to_cpu(lba_status->lsgc));
+
+        elements_list = json_create_array();
+        for (int ele = 0; ele < num_elements; ele++) {
+                element = json_create_object();
+                json_object_add_value_uint(element, "neid",
+                        le32_to_cpu(lba_status->elements[ele].neid));
```

Comment:
I'm terribly confused how this even compiles. We have a variable length array who's elements are variable in size. How could the compiler possibly know the address of the element index?

Reply:
You need -Wpedantic to complain about this. I'm not sure if it is undefined behavior.

Reply:
The behavior assumes the elements are fixed length `sizeof(lba_status->elements[0])`, so accessing anything except for the first index is going to yield the wrong address.

Reply:
I have addressed all the review comments and raised it as separate PR https://github.com/linux-nvme/nvme-cli/pull/933
will close this PR.
Thanks!